auth-web/src/components/auth/common/AccountBusinessType.vue

```diff
@@ -8,7 +14,7 @@
             row
             v-model="isBusinessAccount"
             ref="isBusinessAccount"
-            @change="onOrgBusinessTypeChange"
+            @mouseup="onOrgBusinessTypeChange(true)"
```

Comment:
mouse-up may not work if some one is using tabs?

Reply:
Changed to @change in v-radio buttons. It does the job.